xarray/conventions.py

```diff
@@ -408,6 +408,18 @@ def stackable(dim):
                     new_vars[k].encoding['coordinates'] = coord_str
                     del var_attrs['coordinates']
                     coord_names.update(var_coord_names)
+            if 'bounds' in var_attrs:
+                bounds_str = var_attrs['bounds']
+                var_bounds_names = [bounds_str]
+                if all(k in variables for k in var_bounds_names):
+                    new_vars[k].encoding['bounds'] = bounds_str
```

Comment:
Shifted to be ``encoding`` only.

Reply:
On the other hand, `attrs` seems to stick with the data through more operations, while `encoding` is dropped quickly; this would indicate that `bounds` in `encoding` is for how we save to disk only, while `bounds` in `attrs` is for remembering the relationship while manipulating the data in memory.

Alternately, the mess around where `bounds` should end up could indicate that `bounds` should be dealt with be decoding to a `pd.IntervalIndex` rather than trying to associate the bounds with the coordinate.  Out of curiosity, does `xarray` have a multidimensional generalization of `pd.IntervalIndex` for multidimensional bounds?

Reply:
The CF attributes are now in `encoding` only.